components/authz-service/storage/postgres/migration/sql/63_make_db_id_lookup_functions_strict.up.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
+
+-- lookup by name, the UUID column can go away
+CREATE OR REPLACE FUNCTION member_db_id (
+    _name iam_members.name % TYPE,
+    OUT _db_id iam_members.db_id % TYPE
```

Comment:
Lazy-people feature: it'll resolve to the mentioned column's _type_ at time of the function creation. If that column type changes, it will **not** adapt automatically.